.travis.yml

```diff
@@ -4,12 +4,10 @@ language: python
 
 jobs:
   include:
-    - python: "3.8"
-      env: TOXENV=flake8
     - python: "3.8"
       env: TOXENV=pylint
     - python: "3.8"
-      env: TOXENV=pydocstyle
+      env: TOXENV=lint
```

Comment:
flake8 is now incorporated into pre-commit. We run pre-commit on all files during CI.